app/routes/individual_response.py

```diff
@@ -60,11 +60,15 @@ def before_individual_response_request():
         "individual-response request", method=request.method, url_path=request.full_path
     )
 
-    handle_language()
+    handle_language(metadata)
 
     session_store = get_session_store()
+
     # pylint: disable=assigning-non-slot
-    g.schema = load_schema_from_session_data(session_store.session_data)
+    g.schema = load_schema_from_metadata(
+        metadata=questionnaire_store.metadata,
+        language_code=session_store.session_data.language_code,
+    )
```

Comment:
I would remove the call to session store here and instead use `get_locale().language` which will return the language code from session.